frontend/src/utils/umbra-api.ts

```diff
@@ -10,17 +10,42 @@ import {
   RelayResponse,
   TokenListResponse,
   WithdrawalInputs,
+  UmbraApiVersion,
 } from 'components/models';
 import { jsonFetch } from 'src/utils/utils';
 
+import useSettingsStore from 'src/store/settings';
+
+const { getUmbraApiVersion, setUmbraApiVersion, clearUmbraApiVersion } = useSettingsStore();
+
 export class UmbraApi {
   static baseUrl = 'https://mainnet.api.umbra.cash'; // works for all networks
+  // static baseUrl = 'http://localhost:3000';  // use this for testing with a local Umbra API
```

Comment:
May  be better to move this to comment above the baseUrl variable. Something like `To test locally change to ...`